packages/backend/src/clients/Slack/SlackClient.ts

```diff
@@ -33,7 +33,8 @@ export class SlackClient {
     }: SlackClientArguments) {
         this.lightdashConfig = lightdashConfig;
         this.slackAuthenticationModel = slackAuthenticationModel;
-        this.start();
+
+        void this.start();
```

Comment:
Using the `void` keyword makes it explicit that you are acknowledging this is a promise you don't intend to await or use with `then` or `catch`